package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@ant-design/icons": "^4.6.4",
```

Comment:
@jbarriospd No es buena practica usar el símbolo `^` en prod ya que eso puede hacer que se descarguen versiones actuales de una lib para que el sistema no esta listo, incluso si se tiene el package lock es buena practica tener la versión exacta.

Si puedes de una vez hacer pin a las demás dependencias estaría genial!

Algunas referencias:
https://docs.renovatebot.com/dependency-pinning/
https://the-guild.dev/blog/how-should-you-pin-dependencies-and-why

Reply:
@nicobytes gracias por el feedback ya se encuentran pineadas las dependencias en package.json y package-lock.json